build.sbt

```diff
@@ -432,14 +422,19 @@ lazy val docs =
       paradoxTheme := Some(builtinParadoxTheme("generic")),
       name := "scalapact-docs",
       git.remoteRepo := "git@github.com:ITV/scala-pact.git",
-      sourceDirectory in Paradox := sourceDirectory.value / "main" / "paradox"
+      sourceDirectory in Paradox := sourceDirectory.value / "main" / "paradox",
+      skip in publish := true
     )
 
 lazy val scalaPactProject =
   (project in file("."))
     .settings(commonSettings: _*)
-    .aggregate(shared, core, pluginShared, plugin, pluginNoDeps, framework, standalone, testShared)
+    .settings(
+      skip in publish := true
```

Comment:
What do we need this OOC?

Reply:
SBT by default will try and publish all the things, including the root project which clearly will not publish anything. When it tries to publish the root project with nothing in it, you get an error. There are many flavours of "don't publish this thing" but this method seems to be the only one that works for all modules *including* the root project.